### PR TITLE
vendor containers/psgo@v1.8.0

### PR DESCRIPTION
**Pull request by presztak**
Add two new format-descriptors (`uid`, `huid`) to `top` command.
Also add e2e test for above descriptors.

#### Does this PR introduce a user-facing change?

```release-note
New format descriptors have been added to `podman top` command:
  - 'uid':  prints the effective UID of the process as the decimal representation
  - 'huid':  prints the corresponding host UID of a container process

$ ./bin/podman top 67 pid,uid,huid
PID         UID         HUID
1           0           1000
```
Closes: #16009
Signed-off-by: Piotr Resztak <piotr.resztak@gmail.com>

### CODE REVIEW

File: test/e2e/top_test.go [hunk 1]
```diff
@@ -89,6 +90,12 @@ var _ = Describe("Podman top", func() {
 		result.WaitWithDefaultTimeout()
 		Expect(result).Should(Exit(0))
 		Expect(len(result.OutputToStringArray())).To(BeNumerically(">", 1))
+
+		result = podmanTest.Podman([]string{"top", session.OutputToString(), "uid", "huid"})
+		result.WaitWithDefaultTimeout()
+		Expect(result).Should(Exit(0))
+		Expect(len(result.OutputToStringArray())).To(BeNumerically(">", 1))
```
Comment (vrothberg) [inline test/e2e/top_test.go:97]: Can we run two times top on the container, once with `uid` another time with `huid`?  I really want to have a look at the returned values. `uid` should return 0.  `huid` should return `id -u`.

Comment (presztak) [reply test/e2e/top_test.go:97]: Fixed.

**Review by vrothberg — COMMENTED** [review]
LGTM

Needs a rebase due to the merge conflicts.

**Comment by vrothberg** [comment]
@giuseppe @rhatdan PTAL

**Comment by presztak** [comment]
@vrothberg conflicts resolved.

**Pull request merged**